hugo/data/teams.yml

```diff
@@ -94,6 +76,21 @@ sections:
           name: Denys Smirnov
           position: Senior Software Engineer
           github: https://github.com/dennwc
+
+      - name: Data Processing Team
```

Comment:
Given the ordering rules `Data Processing` should be before `Data Retrieval`

Reply:
Thanks! Done.